src/client/eppo-client.spec.ts

```diff
@@ -936,4 +936,146 @@ describe('EppoClient E2E test', () => {
       );
     });
   });
+
+  describe('flag overrides', () => {
```

Comment:
🙌  Yay Tests! 💪

Reply:
Consider a test if an _existing_ flag is overwritten _after_ an assignment has been made normally. For example it would:
1. request a flag value
2. override that flag
3. request the value (Expect overridden value)

Also we should check what happens when a flag override is made with the wrong type (e.g., override a string flag with a boolean value)